nobl9/resource_agent.go

```diff
@@ -106,11 +116,18 @@ func resourceAgentApply(ctx context.Context, d *schema.ResourceData, meta interf
 	var p n9api.Payload
 	p.AddObject(service)
```

Comment:
```suggestion
	p.AddObject(agent)
```
Could you rename it, please? I guess someone must have copied and pasted the service here :)

Reply:
yes :)